docs/api-testing/use-cases/using-environments.md

```diff
@@ -0,0 +1,142 @@
+:::
+
+<details>
+<summary>
+Code View Example of the test with assertions
+</summary>
+
+```yaml
+- id: get
+  children:
+    - id: queryParam
+      name: api_key
+      value: "1234567890134567890"
+  url: https://run.mocky.io/v3/c99a075d-8e82-4f6c-904f-cac0359bf4a9/api/retail/product/shoes
+  var: payload
+  mode: json
+- id: assert-equals
+  expression: payload_response.headers['Content-Type']
+  value: application/json; charset=UTF-8
+- id: assert-is
+  expression: payload
+  type: array
+- id: each
+  children:
+    - id: assert-is
+      expression: _1.id
+      type: integer
+    - id: assert-exists
+      expression: _1.name
+    - id: assert-exists
+      expression: _1.price
+    - id: assert-exists
+      expression: _1.image
+    - id: assert-exists
+      expression: _1.description
+  expression: payload.pick(5)
+```
+
+</details>
+
+### Step 2: Converting URL into Variables
+
+The test as written above does not allow you to use it in different environments. Therefore, you need to replace the URL and any other data, that are related to the environment the test is running on, into a variable. Now we will do this:
+
+1. Edit **GET** component:
+   - URL - for example `${protocol}${domain}${endpoint}`
+1. In Input Set tab, add a Global Variable:
+   - Name - for example `protocol`
+   - Value - for example `https://`
+1. Add a Global Variable:
+   - Name - for example `domain`
+   - Value - for example `run.mocky.io/v3/c99a075d-8e82-4f6c-904f-cac0359bf4a9`
+1. Add a Global Variable:
+   - Name - for example `endpoint`
+   - Value - for example `/api/retail/product/shoes`
+1. In Unit tab, Edit **QueryParam** component:
+   - Value - for example `${apiKey}`
+1. In Input Set tab, add a Global Variable:
+   - Name - for example `apiKey`
+   - Value - for example `1234567890134567890`
+1. Run the test to confirm the changes do not break the test.
+
+### Step 3: Creating the Staging Environment
+
+In the previous step, we did all the required changes to make the test suitable to run in different environments. Now we will create the `staging` environment for running the test against that.
+
+1. Click **No environments** dropdown list.
+1. Click **Add Item**.
+1. Enter the **Environment name**, for example `staging` then **Confirm**.
+1. Click **Create Variable**:
+   - Key - for example `domain`
+   - Value - for example `run.mocky.io/v3/656dc17e-02ee-43f8-8bad-ed5ce3cc5911`
+1. **Confirm**.
+1. Click **Create Variable**:
+   - Key - for example `apiKey`
+   - Value - for example `09876543210987654321`
+1. **Confirm**.
+1. Click the **Cross** icon.
+   :::info
+   The environment you just created is selected by default in the dropdown list.
+   :::
+1. Run the test with the selected environment.
+
+This time the test has been executed against the `staging` domain with the `apiKey` for staging, instead of the default values (that is `developer environment`)
```

Comment:
```suggestion
This time the test has been executed against the `staging` environment with the `apiKey` for staging, instead of the default value (that is `developer` environment)
```